### ISSUE TITLE
Webdesign: Sitemap Feedback 

### ISSUE THREAD

**Issue by KejoBuchanan** [CivicTechWR/WRvotes#17]
Please let me know feedback and content suggestions.

**Comment by pnijjar** [comment]
There is apparently a referendum in Cambridge. I do not know whether this belongs in the main menu or not. 

Is there a single page showing 2014 results? I am thinking the core of the site is a page that shows information for a single ward: 

- The races applicable to that ward, and the candidates running there
- Upcoming events for races in that ward
- Archived resources for that ward (special interest questionnaires, all-candidates recordings)
- News coverage within that ward

We could perhaps include 2014 results there. 

It is also not clear to me whether we should have much information for candidates on the site, but I could be wrong about this. 

What would go in the social media section? Would that be better information to include in the footer for the site?

**Comment by KejoBuchanan** [comment]
Apologies for the confusion this sitemap lists the content and parent/child navigation.   A wireframe will better explain the placement of all the features on the page.  So think of this as a placecard test to see if we have the right headers for the right things, as well as edit/improve naming so it is understood by our primary users.

I have edited according to your suggestion to rename "2014 election summary" to "2014 Results".  We can always archive content if it starts to get cluttered or become removed from our main goal.

Currently, our three main menu/navigation bar are **About, Candidates, Resources & Events**.  The other two are content that will appear somewhere on the page along with the home button/icon in the header.

Thank you for the feedback, keep it coming everyone :)

**Comment by pnijjar** [comment]
I could see the version of the flowchart you linked to in Slack, but not the link here. This one wants me to authenticate to Google Drive.

**Comment by KejoBuchanan** [comment]
Yes it's a link for anyone to connect through Google Drive and edit.
